assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -42,4 +43,18 @@ export const tests: Tests = {
     independent: true,
     seed: 5,
   },
+
+  gbStructureTest: {
+    variants: ['control', 'contributeOnTop'],
+    audiences: {
+      GB: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: false,
+    independent: true,
+    seed: 5,
```

Comment:
Do we not want a different seed from `usRecurringAmountsTest`?

Reply:
Yeap!